subprojects/docs/src/docs/release/notes.md

```diff
@@ -134,6 +134,26 @@ You can now use the `export-keys` flag to export all already trusted keys:
 
 For more information, see [Exporting keys](userguide/dependency_verification.html#sec:local-keyring).
 
+#### Updated the Kotlin DSL to Kotlin API Level 1.7
+
+Previously, the Kotlin DSL used Kotlin API level 1.4.
+Starting with Gradle 8.0, the Kotlin DSL uses Kotlin API level 1.7.
+This change brings all the improvements made to the Kotlin language and standard library since Kotlin 1.4.0.
+
+For information about breaking and nonbreaking changes in this upgrade, see the following links to the Kotlin documentation:
```

Comment:
We could consider removing that block, have the highlights and then at the end link to the upgrade guide for potential breaking changes.

Reply:
Good point! Done